docs/developers.md

```diff
@@ -64,14 +63,14 @@ If you wish to create a Java JAR package, which can then be run without [Gradle]
 ```
 
 which will create `logisim-evolution-<version>-all.jar` in `build/libs/`. To use this package
```

Comment:
Isn't there a comma missing after "To use this package"?

Reply:
Fixed

Reply:
How about: "To run it with JRE/JDK 14 or higher, type"?